server/src/utils/callAutomationUtils.ts

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Suggestion. Would be great to have in the readme how this works if a developer is going to make an update to it.
There's async events from the websockets, various STORE_MAPS and its really hard to piece everything together reading the code as its not a linear feature.
Some sort of flowchart diagram would be very helpful

Reply:
absolutely. This will come in a future PR closer to the end.